complete-upload/src/main/java/no/unit/nva/fileupload/CompleteUploadHandler.java

```diff
@@ -32,6 +32,12 @@ public class CompleteUploadHandler extends ApiGatewayHandler<CompleteUploadReque
     public static final String S3_ERROR = "S3 error";
     public static final String FILE_NAME_REGEX = "filename=\"(.*)\"";
 
+    // The total time the SDK will wait for the entire request execution, including retries
+    public static final int SDK_CLIENT_EXECUTION_TIMEOUT = 8 * 1000;
+
+    // The time the SDK will wait for data transfer for a single request.
+    public static final int SDK_REQUEST_TIMEOUT = 2 * 1000;
```

Comment:
Should we include the time unit in these constants, and perhaps also add a comment with the total?